src/components/CartAside/CartAside.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useRef } from 'react';
+import { connect } from 'react-redux';
+
+import { CartAsideItem } from './CartAsideItem';
+import './CartAside.scss';
+
+const CartAside = ({ cartItems }) => {
+
+   const [cartStatus, setCartStatus] = useState(false);
+   const cartAsideRef = useRef('');
+   const toggleStatus = cartStatus ? 'cart-state-open' : 'cart-state-closed';
+
+   const asideToggleHandler = () => {
+      cartStatus ? setCartStatus(!cartStatus) : setCartStatus(!cartStatus);
```

Comment:
Useless ternary operator
```suggestion
      setCartStatus(!cartStatus);
```